core/object/object.h

```diff
@@ -51,6 +51,7 @@ enum PropertyHint {
 	PROPERTY_HINT_ENUM, ///< hint_text= "val1,val2,val3,etc"
 	PROPERTY_HINT_ENUM_SUGGESTION, ///< hint_text= "val1,val2,val3,etc"
 	PROPERTY_HINT_EXP_EASING, /// exponential easing function (Math::ease) use "attenuation" hint string to revert (flip h), "positive_only" to exclude in-out and out-in. (ie: "attenuation,positive_only")
+	PROPERTY_HINT_HIDE_QUATERNION_EDIT, /// Only Node3D::transform should hide the quaternion editor.
```

Comment:
Any particular reason why it's added in the middle, and not at the end? I'm also not a fan of having negative hint like that, personally.

Reply:
> I added this near them because it is a hint on how to set the value, as different from a hint on selecting a resource type etc. It was just the middle of the Enum list as a result.

I don't see it being any different than `PROPERTY_HINT_MULTILINE_TEXT` for example, which just changes how the inspector widget looks, same as your change. But more to the point, we don't have any sensible grouping for these, so there isn't really any order, so I'd say it is preferable to add things to the end of the list to avoid unnecessary conflicts.

We do want to reorder them at some point. If we manage to do it before the beta, I guess. So this can be revisited then.

Reply:
Okay, replaced it to end of list now.